src/app/static/src/app/store/translations/locales.ts

```diff
@@ -95,6 +97,7 @@ const en: Translations = {
     cancelRun: "Cancel run",
     chooseFile: "Choose a file",
     clickHere: "Click here",
+    colorIndicator: "Colour Indicator",
```

Comment:
I'm assuming here that we're using UK English in the UI, but the code uses 'color' elsewhere so would be better for the key to be consistent with that, hence this slight oddity